cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/transaction/TransactionRunners.java

```diff
@@ -161,9 +181,9 @@ private static RuntimeException propagate(Throwable throwable) {
    * throw the failure exception as-is the given propagated type if the type matches or as {@link RuntimeException}.
    * This method will always throw exception and the returned exception is for satisfying Java static analysis only.
    *
-   * @param e the {@link TransactionException} to propagate
-   * @param propagateType if the exception is an instance of this type, it will be rethrown as is
-   * @param <X> exception type of propagate type
+   * @param e the {@link TransactionException} to propagate.
```

Comment:
please remove all the preriods in the @param, @return and @throws annotations.

Reply:
I should have been more precise. If the @param annotation is multiple sentences, then it needs periods at the end of each sentence, including the last sentence. If it is only one sentence, actually it is always just a fragment of a sentence, and therefore should not have a period.